docs/get-started/actions/checkLink.md

```diff
@@ -79,4 +75,96 @@ Here are a few ways you might you the `checkLink` action:
     }
   ]
 }
-```
\ No newline at end of file
+```
+
+## Troubleshooting
+
+- [`checkLink` fails due to unrecognized certificates](#checklink-fails-due-to-unrecognized-certificates)
+
+### `checkLink` fails due to unrecognized certificates
+
+If the `checkLink` action fails for a URL that is valid and loads without redirects, it may be due to an internal or custom certificate that is not recognized by the testing machine.
```

Comment:
```suggestion
If the `checkLink` action fails for a valid URL that loads without redirects, it may be due to an internal or custom certificate that the testing machine does not recognize.
```